releasenotes/README.md

```diff
@@ -20,15 +20,17 @@ area: traffic-management
 issue:
   - https://github.com/istio/istio/issues/23622
   - 23624
-releaseNotes: |
+releaseNotes: 
+- |
```

Comment:
Do such one-line entries need the pipe?

Reply:
The readme is an example of how they should look, so I think it's fine.